aspnetcore/Controllers/CheckoutController.cs

```diff
@@ -1,48 +1,37 @@
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
-using Microsoft.AspNetCore.Mvc;
-using Microsoft.Extensions.Logging;
-using aspnetcore.Models;
-using Microsoft.Extensions.Configuration;
-using Sentry;
+namespace Empower.Backend.Controllers;
 
-namespace aspnetcore.Controllers
+[ApiController]
+[Route("[controller]")]
+public class CheckoutController : ControllerBase
 {
-    [ApiController]
-    [Route("[controller]")]
-    public class CheckoutController : ControllerBase
+    [HttpPost]
+    public void Checkout()
     {
-        private readonly ILogger<CheckoutController> _logger;
-        private hardwarestoreContext _context = null;
-        private readonly IConfiguration Configuration;
+        var se = (string?) Request.Headers["se"];
```

Comment:
Would we want this set in all events? e.g: adding this on a middleware

Reply:
Not sure.  I didn't investigate the end-to-end usage of the application.  The endpoints all do what they previously did.  There's still room for improvement such as this, or making an outbound http request, or connecting some of the other endpoints to return results from the database.